redi-connect-front/src/pages/front/signup/signUpFormFactory.tsx

```diff
@@ -70,7 +70,7 @@ export interface SignUpFormValues {
 
 const initialValues: SignUpFormValues = {
   formType: 'public-sign-up-mentee-pending-review',
-  gaveGdprConsentAt: '',
+  gaveGdprConsent: false,
```

Comment:
This value, `gaveGdprConsent`, is currently submitted to the server on form completion (i.e. as part of line 104. Please add this property to the list on lines 94-99, i.e. of `RedProfile` properties to omit).

Reply:
done

Reply:
Thanks!